packages/core/admin/admin/src/constants.ts

```diff
@@ -125,7 +125,8 @@ export const HOOKS = {
 
 export interface SettingsMenuLink
   extends Omit<StrapiAppSettingLink, 'Component' | 'permissions' | 'lockIcon'> {
-  lockIcon?: boolean;
+  // TODO: to replace with another name in v5
+  lockIcon?: boolean; // TODO: to replace with another name in v5
```

Comment:
Maybe we can remove one of those comments 😅

Reply:
Also, aren't we merging this in v5? Is it something we want to address now or in another dedicated PR?

Reply:
ahh yeah ok. I took a look at all this with @simotae14 and he said to accept what was currently on v5/main since what is on develop depends on https://github.com/strapi/strapi/pull/20431 to be merged first.

Reply:
these comments are just a reminder for me, I need to replace the Lock icon with the new Lightning one and also change the property name lockIcon to something with more sense

Reply:
but for sure you can remove one of those comments